bin/bump-version

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Automatically updates the tap for a newly-released version of materialized.
+
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+  echo "usage: $0 VERSION" >&2
+  exit 1
+fi
+
+cd "$(dirname "$0")/.."
+
+version=${1#v}
+
+echo "Fetching metadata for materialized v$version..."
+url=https://github.com/MaterializeInc/materialize/archive/v$version.tar.gz
+archive_sha=$(curl -fsSL "$url" | openssl sha256)
+build_sha=$(git ls-remote https://github.com/MaterializeInc/materialize.git "v$version^{}" | cut -f1)
+
+echo "URL:                    $url"
+echo "Source archive SHA-256: $archive_sha"
+echo "Build commit SHA-1:     $build_sha"
+
+echo "Updating formula for new source tarball..."
+sed -f /dev/stdin Formula/materialized.rb > Formula/materialized.rb.new <<EOF
+/^  url/c\\
+\  url "$url"
+/^  sha256/c\\
+\  sha256 "$archive_sha"
+/^  STABLE_BUILD_SHA/c\\
+\  STABLE_BUILD_SHA = "$build_sha".freeze
+EOF
+mv Formula/materialized.rb{.new,}
```

Comment:
would it be more straightforward to just have a template file that we sed into place?

doing `sed -e "s/MZ_VERSION/$mz_version/"  -e "s/MZSTABLE_BUILD_SHA/$stable_sha/" .. Build.template.rb > Formula/materialized.rb` seems less fragile and easier to debug (since you can just grep for the strings).

Reply:
Ah, maybe, but there are some complexities to work out there too because you need to render _twice_, once without the bottle stanza and once with. Going to merge as is for now, but if you figure out how to make it cleaner I'm all for it.